### ISSUE TITLE
DEPRECATION: Using the injected `container` is deprecated. Please use the `getOwner` helper instead to access the owner of this object.

### ISSUE THREAD

**Issue by balos1** [BookingSync/ember-data-partial-model#20]
It looks like starting with Ember 2.1 accessing the container directly is deprecated. Because of this, when I load about ten thousand records, ten thousand deprecation errors occur, which bogs down the loading significantly (and if inspect is open, it even crashes chrome from the console logging). I think the offending code is found in the [store mixin](https://github.com/BookingSync/ember-data-partial-model/blob/master/addon%2Fmixins%2Fstore.js).

```
function _getRegistry(store) {
  // support pre Ember 2.1.x (Ember 2.0.x, 1.13.x)
  if (store.container._registry) {
    return store.container._registry;
  } else {
    return store.container.registry;
  }
}
```

The Ember version I am using is 2.3.0-beta-3 and the Ember-Data version in 2.3.0-beta-4.

**Comment by Azdaroth** [comment]
@cojomojo  Thanks for mentioning the issue, this needs to be migrated to new Owner API, we will take care of it this week.

**Comment by Azdaroth** [comment]
Hi @cojomojo, we've just released version 0.3.1, you may want to check it out, all warnings should be gone now.
